css/index.css

```diff
@@ -85,37 +88,101 @@ p {
     padding-bottom: 20px;  
 }
 
-.middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+h2{
+    font-weight: bolder;
 }
-
-.middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+/*BOXES MIDDLE CONTENT*/
+.boxcontainer div{
+    margin-top:5%;
+    display:flex;
+    flex-direction: row;
+    justify-content:space-evenly;
 }
 
-.middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
+/* .box{
+    width:20%;
+    border:3px solid black;
+    text-align:center;
+    padding:4%;
+   
+    
+} */
+
+.box1{
+    width: 100px;
+    height: 100px;
+    border:3px solid black;
+    background-color: teal;
     justify-content: space-evenly;
+    /* display: flex;
+    flex-direction: row; */
 }
 
```

Comment:
The parent container for each individual box can take care of all the styling needed for each box.  You then want to just override what needs to change for box1, box2 etc.  In this case the background color.